templates/datapipeline.yml

```diff
@@ -109,6 +109,12 @@ Resources:
                 fi
                 sudo aws s3 sync --delete --exact-timestamps /backup/ s3://$destination/
```

Comment:
Didn't you want to add something about `symlinks`? Or add support for extra args?

Reply:
Never mind.

> Return code 2 means that the following were skipped: files/symlinks that do not exist, files that are character special devices, block special device, FIFO's, or sockets, and files that the user cannot read from. We need to return 0 in this case for the pipeline to succeed because all normal files were synched successfully